src/Apps/Settings/Routes/MyCollection/Components/MyCollectionAppDownload.tsx

```diff
@@ -8,71 +8,106 @@ import {
   Spacer,
   Text,
 } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
 import { resized } from "Utils/resized"
+import { Media } from "Utils/Responsive"
 
-export const MyCollectionAppDownload: React.FC = () => {
-  const image = resized(
-    "https://files.artsy.net/images/my-collection-get-app-img.jpg",
-    {
-      width: 910,
-      height: 652,
-    }
+const image = resized(
+  "https://files.artsy.net/images/my-collection-get-app-img.jpg",
+  {
+    width: 910,
+    height: 652,
+  }
+)
+
+const AppStoreBadges = () => (
+  <Flex>
+    <RouterLink
+      to="https://apps.apple.com/us/app/artsy-buy-sell-original-art/id703796080"
+      target="_blank"
+    >
+      <Box width={120} height={40}>
+        <Image
+          src="https://files.artsy.net/images/download-ios-app.svg"
+          width="100%"
+          height="100%"
+          alt="ios app store button"
+          loading="lazy"
+        />
+      </Box>
+    </RouterLink>
+
+    <Spacer ml={2} />
+
+    <RouterLink
+      to={"https://play.google.com/store/apps/details?id=net.artsy.app"}
+      target="_blank"
+    >
+      <Box width={120} height={40}>
+        <Image
+          src="https://files.artsy.net/images/download-android-app.svg"
+          width="100%"
+          height="100%"
+          alt="android play store button"
+          loading="lazy"
+        />
+      </Box>
+    </RouterLink>
+  </Flex>
+)
+
+const Title = () => {
+  const title = "Get More from My Collection on the App"
+  return (
+    <>
+      <Media greaterThanOrEqual="lg">
+        <Text variant="xxl" mt={1}>
```

Comment:
`variant` is a responsive prop: `variant={['xxl', 'lg]}` this usage of `Media` is unnecessary. It's extraction into its own component is also unnecessary and results in an isolated external margin.

Reply:
Updated